test/utils/document.js

```diff
@@ -0,0 +1,41 @@
+/* global window */
+
+export default class DocumentTest {
```

Comment:
Add JSDoc for this class.

Reply:
The class name `DocumentTest` does not communicate what it does, e.g. the creation of a canvas. When you write tests using this class, it will be difficult for other maintainers to understand a) what the callback receives b) what the callback is supposed to return, from the code `new DocumentTest(...)`.